recipes/scikit-sundae/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "scikit_sundae" %}
+{% set version = "1.0.0rc2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 644c5518203bc935c0ce242db004747caed53fe80ba6dd5ca2c550cfb2a82978
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy
+    - sundials >=7.1,<7.2
```

Comment:
I made the recommended change, but it caused all builds to fail. Seems like they are indeed needed as build dependencies. However, although I added them back, I am now failing on OSX builds (which is weird because they were passing before). The error seems to be coming from conda/core/index.py and gets thrown before the package starts building:

Traceback (most recent call last):
  File "/Users/runner/miniforge3/lib/python3.12/site-packages/conda/core/index.py", line 310, in data
    return self._data
           ^^^^^^^^^^
AttributeError: 'Index' object has no attribute '_data'. Did you mean: 'data'?

Please advise on how to get this resolved. Thanks for your support!

Reply:
Seems to be related to a breaking change from elsewhere. A solution is being pursued in Issue #28242. I will re-run the builds after the issue is resolved.